docs/excel/custom-functions-overview.md

```diff
@@ -342,11 +342,11 @@ function secondHighest(values){
 
 ## Discovering cells that invoke custom functions
 
-Custom funtions also allows you to format ranges, display cached values, and reconcile values using `caller.address`, which makes it possible to discover the cell that invoked a custom function. You might use `caller.address` in some of the following scenarios:
+Custom funtions also allows you to format ranges, display cached values, and reconcile values using the address of the cell which invokes a custom function. You might use this functionality in some of the following scenarios:
 
-- Formatting ranges: Use `caller.address` as the key of the cell to store information in [AsyncStorage](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [onCalculated](https://docs.microsoft.com/javascript/api/excel/excel.worksheet#oncalculated) in Excel to load the key from `AsyncStorage`.
+- Formatting ranges: Use the cell's address as the key to store information in [AsyncStorage](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [onCalculated](https://docs.microsoft.com/javascript/api/excel/excel.worksheet#oncalculated) in Excel to load the key from `AsyncStorage`.
```

Comment:
Since this section is about "discovering" the address, shouldn't the caller.address info be included?

Reply:
So it turns out that caller.address doesn't actually exist as an object. It used to, but in this latest version, it isn't a thing. It's just been shorthand for folks when talking about this feature. Keyur asked me to change this section so that any mention of caller.address is taken out and instead use the "address of the cell" as a different way to describe it.

Reply:
got it thanks!